laborator/content/linking/demo/static/static-2.c

```diff
@@ -0,0 +1,12 @@
+/*
+ * Author: Gabriel Mocanu <gabi.mocanu98@gmail.com>
+ */
+
+#include <stdio.h>
+
+int main()
+{
+    static int x;
```

Comment:
Ar merge un comentariu in care sa spui ca e in `.bss`, drept care e initializat cu 0.

Reply:
Am pus. Nu zic ca e initilizat cu 0 ca asta era de fapt intrebarea la exemplul asta.